### PR TITLE
David/small fix and parallelizing

### PR DESCRIPTION
**Pull request by davidjoy**
Didn't feel like making two separate branches.

1. Fixes an error in how platform names are given to StaticListItem.
2. Parallelizes fetch profile calls.

### CODE REVIEW

File: src/components/ProfilePage/SocialLinks.jsx [hunk 1]
```diff
@@ -26,10 +25,12 @@ class SocialLinks extends React.Component {
   }
 
   onSave() {
-    const values = this.props.platforms.filter(({ key }) => typeof this.state[key] !== 'undefined').map(({ key }) => ({
-      platform: key,
-      socialLink: this.state[key],
-    }));
+    const values = this.props.platforms
+      .filter(({ key }) => typeof this.state[key] !== 'undefined')
```
Comment (albemarle) [inline src/components/ProfilePage/SocialLinks.jsx]: Could also be !_.isUndefined(this.state[key])

(The lack of _.isDefined() is a perpetual sorrow to me.)

File: src/components/ProfilePage/SocialLinks.jsx [hunk 2]
```diff
@@ -190,16 +192,15 @@ SocialLink.propTypes = {
...
-  platform,
-  onClickEmptyContent,
-  name,
+  url, platform, onClickEmptyContent, name,
 }) {
-  const linkDisplay = url != null ?
-    <SocialLink name={name} url={url} platform={platform} /> :
-    <EmptyContent onClick={onClickEmptyContent}>Add {name}</EmptyContent>;
+  const linkDisplay =
+    url != null ? (
```
Comment (albemarle) [inline src/components/ProfilePage/SocialLinks.jsx]: Philosophical note about two things in tension here: the case where it's not null is the one you care more about, so it makes sense to put it first, but on the other hand, conditionals are easiest for humans to read if they're not unpositive.  I tend to prioritize the second consideration more than the first.

**Comment by abutterworth** [comment]
Heads up I have a reworked version of social links in a different PR.

**Comment by davidjoy** [comment]
This PR has been subsumed by #40.  Closing.

**Pull request closed without merging**